### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]: "Is Toggle pattern" is False for listitems in "Select Subscriptions" dialog

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1393]
**Repro Steps:**

1. Open and Start Eclipse.
2. Navigate to Tools menu-> "Azure"-> "Azure Sign In"
3. In "Azure Sign in" dialog, tab to and select "Automated" radio button.
4. Tab to and select "Browse" button.
5. In "Create Authentication Files" dialog , select the required subscriptions and select "Start" button.
6. In "Service Principal Creation Status dialog", tab to and select "OK" button.
7. In "Your Subscriptions" dialog , tab to list items and observe the "Is Toggle Pattern" using inspect

**Actual Result:**
"Is Toggle Pattern" is False for listitems in "Select Subscriptions" dialog.

**Narrator:**
CIS_BVT_Helsinki, <say-as interpret-as="spell">1</say-as> of 12, selected, 
Space
Space

**NVDA:**
list
CIS_BVT_Helsinki  check box  selected  not checked  Subscription ID: 0bababbd-f605-4b9e-889e-84cc5b966025
CIS_BVT_Helsinki  check box  checked  selected  Subscription ID: 0bababbd-f605-4b9e-889e-84cc5b966025

**Expected Result:**
"Is Toggle Pattern" should be true for list items in "Select Subscriptions" dialog.
Narrator and NVDA should announce as
CIS_BVT_Helsinki, <say-as interpret-as="spell">1</say-as> of 12, selected, unchecked
checked
unchecked

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![select subscription-inspect](https://user-images.githubusercontent.com/22956130/35683351-a8bd75fe-0789-11e8-8c15-c77b13d89e70.jpg)
![select subscription-narrator](https://user-images.githubusercontent.com/22956130/35683359-ac31758c-0789-11e8-8495-0b12306ac617.jpg)
![select subscription-nvda](https://user-images.githubusercontent.com/22956130/35683365-af2bc2ec-0789-11e8-8ea1-b96420e403b5.jpg)

**Comment by Eskibear** [comment]
fixed.

**Comment by Eskibear** [comment]
shipped in v3.6.0

**Comment by WiproPSAccTest** [comment]
Verified the issue in V3.6.0.
Issue still exists
![1393-not fixed](https://user-images.githubusercontent.com/22956130/39797061-bfb03258-5377-11e8-9a7b-9658e3af2bf6.jpg)

**Comment by Eskibear** [comment]
The table is created using SWT's table widget, with option `SWT.CHECK` enabled. That's the widget's behavior which we cannot change.

**Comment by Tanushagottipati** [comment]
This issue is still reproducing in the current test pass

**Comment by Tanushagottipati** [comment]
This issue is still reproducing in the current test pass

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749844
